dkms.in

```diff
@@ -1062,6 +1062,12 @@ prepare_build()
     done
 
     if [[ -e "${kernel_config}" ]]; then
+        local cc=$(sed -n 's|^CONFIG_CC_VERSION_TEXT="\([^ ]*\) .*"|\1|p' "${kernel_config}")
+        if command -v "$cc" >/dev/null; then
+            export CC="$cc"
+            export KERNEL_CC="$cc"
+        fi
+
         if grep -q 'CONFIG_CC_IS_CLANG=y' "${kernel_config}"; then
```

Comment:
The hunk was removed by @hadogenes with 4729aefebb98d229e542609c2b6c92545c7a69d8. Seems correct at a glance.

@hadogenes can you confirm this doesn't regress on your end? Which Suse variant/version are you using - SLES, Leap, Tumbleweed? Thanks

Reply:
clang detection is probably wrong if versioned clang-xx is to be used, but as long as we don't have a clang compiled kernel  in Debian, I'm not going to investigate